### PR TITLE
417 name internal vars with an underscore

### PR DESCRIPTION
**Pull request by lirona**

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ module.exports = {
     },
     goerli: {
       url: `https://goerli.infura.io/v3/${process.env.INFURA_KEY}`,
-      accounts: [process.env.GOERLI_PK],
+      accounts: process.env.GOERLI_PK,
```
Comment (ben-kaufman) [inline hardhat.config.js:53]: Why was this changed here (also why in other networks no)?

Comment (ben-kaufman) [reply hardhat.config.js:53]: Looks like the change breaks CI...

Comment (jellegerbrandy) [reply hardhat.config.js]: it does fixing it
